.github/workflows/pull_request.yml

```
@@ -0,0 +1,20 @@
+name: Pull request
+
+on:
+  pull_request:
+    types: [opened, reopened, synchronize]
+
+jobs:
+  tests:
+    name: Test
+    uses: swiftlang/github-workflows/.github/workflows/swift_package_test.yml@main
+    with:
+      linux_swift_versions: "[\"nightly-main\", \"nightly-6.2\"]"
+      windows_swift_versions: "[\"nightly\", \"nightly-6.2\"]"
```

I think we should just do `nightly-main` and `nightly` for now. 6.2 isn't going to work, in the short term.

```suggestion
      linux_swift_versions: "[\"nightly-main\"]"
      windows_swift_versions: "[\"nightly\"]"
```